src/komet/kdist/soroban-semantics/host/context.md

```diff
@@ -32,6 +32,21 @@ module HOST-CONTEXT
 
 ```
 
+## contract_event
+
+```k
+    rule [hostfun-contract-event]:
+        <instrs> hostCall ( "x" , "1" , [ i64  i64  .ValTypes ] -> [ i64  .ValTypes ] )
+              => toSmall(Void)
```

Comment:
I'd suggest adding TODO comments on points that we might need to revisit in the future. Since we are not doing anything with events right now, we may eventually need to change them.